.github/workflows/ci.yaml

```diff
@@ -143,6 +143,9 @@ jobs:
                       folder: e2e/bzlmod
                     - bzlmodEnabled: false
                       folder: e2e/bzlmod
+                    # Don't run workspace smoke test under bzlmod
```

Comment:
just moving these lines higher?

Reply:
Yep just clarifying that it should not be run, rather than to be converted later.